spec/features/staff/fund_managers_can_edit_an_activity_spec.rb

```diff
@@ -45,6 +28,22 @@
     end
 
     context "when the activity is complete" do
+      it "editing and saving a step returns the user to the activity page" do
+        activity = create(:activity, organisation: organisation)
+        identifier = "AB-CDE-1234"
+        visit organisation_activity_path(activity.organisation, activity)
+
+        within(".identifier") do
+          click_on("Edit")
```

Comment:
This feels so fussy but I'd rather this was `I18n.t("generic.link.edit")` than the string Edit, to keep it consistent!